src/chat/discord.md

```diff
@@ -1,4 +1,104 @@
+Select the expansion arrow next to the server name banner (titled "The Rust Programming Language") and select Notifications from the dropdown. Then follow the configuration instructions provided [on the Discord Support site](https://support.discordapp.com/hc/en-us/articles/215253258-Notifications-Settings-101).
+
+# Appropriate conversation
+Discussions should be related to the channel purpose. On team channels, conversation should be related to team business.
+All channels are expected to be used for purposes related to the Rust project. Discussion of (for example) wildlife or sightseeing are not appropriate. Discord, like all official Rust spaces, is governed by the [Code of Conduct](https://www.rust-lang.org/community#conduct).
+
+# Channels
+
+The following channels are especially relevant to newcomers to the Rust project: 
+* rustlings - If you are learning Rust using the [Rustlings tutorials](https://github.com/rust-lang/rustlings), this channel is a great place to meet your fellow students. 
+* rust-usage - This is a channel where you can access support for resolving specific language use questions. The [Rust Users Forum](https://users.rust-lang.org/) is also relevant to your needs.
+* beginners - Here, you can meet people who began using Rust relatively recently. 
+* contribute - Interested in contributing to the Rust project? In addition to joining the discussion on this channel, you can subscribe to the newsletter and learn more about project teams.  
+    * Newsletter <br/>
+Contribution opportunities are regularly posted in the [This Week In Rust](https://this-week-in-rust.org/) newsletter. 
+    * Team descriptions <br/>
+Find out more about [specific teams](https://www.rust-lang.org/governance#teams), including those with channels included in the list below.
+
+*Channel list*
+
+GENERAL
+* rust-usage
+* beginners
+* contribute
+
+COMMUNITY TEAM
+* content-team
+* events
+* community-team
+* website
+* rustbridge
+* marketing
+* localization
+
+LANG TEAM
+* design
+* compiler
+* wg-grammar
+* wg-compiler-performance
+* wg-async-foundation
+* type-theory
+
+OPS
+* infra
+* release
+* triage-wg
+* rollups
+* crater
+* rfcbot
+* crates-io
+
+DEV-TOOLS TEAM
+* dev-tools
+* ides-and-editors
+* rustdoc
+* wg-debugging
+* cargo
+* wg-rustfix
+* clippy
+* wg-rustfmt
+* wg-testing
+* wg-rustup
+* wg-cargo-src
+* wg-bindgen
+
+DOMAIN WORKING GROUPS
+* wg-embedded 
+* wg-cli
+* wg-wasm
+* wg-governance
+* wg-gamedev
+
+DOCUMENTATION TEAM
+* docs
+* rustlings
+* reference
+
+# Messages
+Discord is used asynchronously as well as synchronously, so you should not generally expect that your messages will receive a response quickly unless a meeting is taking place. Depending on how your notifications are configured, you will see a red circle on top of the Discord icon in your system tray when new messages are received. If you wish to communicate with a specific individual, right-click on their user icon and select "Message" in the dropdown menu.
```

Comment:
I'd use less technical phrasing here. "Discord activity depends on when people are available, so..."

Reply:
Revised to address this comment. Thanks!